pkg/validate/streaming.go

```diff
@@ -189,6 +190,12 @@ func checkExec(c internalapi.RuntimeService, execServerURL, stdout string, stdou
 	e, err := remoteclient.NewSPDYExecutor(&rest.Config{TLSClientConfig: rest.TLSClientConfig{Insecure: true}}, "POST", url)
 	framework.ExpectNoError(err, "failed to create executor for %q", execServerURL)
 
+	we, err := remoteclient.NewWebSocketExecutor(&rest.Config{TLSClientConfig: rest.TLSClientConfig{Insecure: true}}, "GET", url.String())
+	framework.ExpectNoError(err, "failed to create websocket executor for %q", execServerURL)
+
+	e, err = remoteclient.NewFallbackExecutor(we, e, httpstream.IsUpgradeFailure)
+	framework.ExpectNoError(err, "failed to create fallback executor for %q", execServerURL)
+
```

Comment:
This is making the tests fail in the CI. Do runtimes have to support that?

Reply:
If websocket failed, will fallback to spdy that is same to before. Why make ci failed?
I think that IsUpgradeFailure will ignore error of websocket upgrade

Reply:
just remove this first, wait runtimes support it